spec/lib/diaspora_federation/validators/comment_validator_spec.rb

```diff
@@ -15,14 +15,6 @@ module DiasporaFederation
       end
     end
 
-    %i(author_signature parent_author_signature).each do |prop|
-      describe "##{prop}" do
-        it_behaves_like "a property that mustn't be empty" do
-          let(:property) { prop }
-        end
-      end
-    end
-
```

Comment:
Please add a `it_behaves_like "a relayable validator"` where you still test this.